Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -11,10 +11,14 @@ include("utils.functions.filesystem.files");
 */
 Wine.prototype.DXVK = function (dxvkVersion) {
     var operatingSystemFetcher = Bean("operatingSystemFetcher");
+    var uiQuestionFactory = Bean("uiQuestionFactory");
     print("NOTE: wine version should be greater or equal to 3.10");
-    if (operatingSystemFetcher.fetchCurrentOperationSystem() != "Linux")
+    if (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() !== "Linux")
     {
-        this.wizard().message(tr("DXVK might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement"));
+        uiQuestionFactory.create(
+				tr("DXVK is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Do you want to continue? Chosing no will delete the wineprefix.", winePrefix),
+				() => remove(this.winePrefixesDirectory + winePrefix)
```

Comment:
Be careful with the copy-paste ! Just skip the installation of DXVK !

Reply:
I'm not sure how. The break command or throw?

Reply:
You use a big if else with a continue var, which can be set true or false depending on the result.